File: hw2/Codes/Oztunc_Zeynep.php
<?php //php 7.2.24

//example of an unconditional exit using break
$loopVar = 10;

while($loopVar >= 1) {
  $loopVar--;
    break;
}
echo "The value of loopVar after the loop is: $loopVar \n";

//example of using continue
$loopVar = 10;

while($loopVar >= 1) {
  $loopVar--;
   continue;
}
echo "The value of lopoVar after the loop is: $loopVar \n";

//example of breaking the outer loop using break
//the outer for loop
for ($i = 0; $i < 5; $i++) {
   echo "\n Inside the outer loop, i:  $i \n" ; 
   print("------------------------------ \n");

   //the inner for loop
   for ($j = 6; $j < 9; $j++) {
      if ($i == 2) {
        echo "Exiting \n";
        break 2; //terminates the 2nd outer loop
      }
      echo "Inside the inner loop, j:  $j \n" ; 

   }
   echo "\n";
} 

//example of using continue

echo "\n Using continue \n" ; 

//the outer for loop
for ($i = 0; $i < 5; $i++) {
   echo "\nInside the outer loop, i:  $i \n" ; 
   print("------------------------------ \n");

   //the inner for loop
   for ($j = 6; $j < 9; $j++) {

      if ($i == 2) {
        echo "Continuing from outer \n";
        continue;  //continues from the outer loop
      }
          echo "Inside the inner loop, j:  $j \n" ; 
       
   }
   echo "\n";
}
?>

File: hw2/Codes/Oztunc_Zeynep.rs
//rustc 1.39.0
fn main() {
    //example of unconditional exit using break
   let mut loopVar = 10;
   while loopVar >= 1{
      loopVar-=1;
      break;
   }
   println!("The value of loopVar after the loop is {}",loopVar);
    
   //example of using continue
   let mut loopVar = 10;
   while loopVar >= 1{
       loopVar-=1;
       continue;
   }
   println!("The value of loopVar after the loop is {}",loopVar);
    
	//example of break in enclosing loops
    //outer label
    'outer: 
        for i in 0..5 {
        println!("Inside the outer loop, i : {}", i);
        println!( "------------------------------");

        //inner label
        'inner:    
        for j in 6..9 { 
            
            //condition for breaking
            if(i==2){
               println!("Exiting\n");
               break 'outer;
            }
            println!("Inside the inner loop, j : {}", j);

        }
        println!( "\n");
    }
    
    println!("Example of continue \n");
    //example of continue in enclosing loops
    //outer label
    'outer: 
        for i in 0..5 {
        println!("Inside the outer loop, i : {}", i);
        println!( "------------------------------");

        //inner label
        'inner:    
        for j in 6..9 { 
            
            //condition for continuing
            if(i==2){
               println!("Continuing");
               continue 'outer;
            }
            println!("Inside the inner loop, j : {}", j);

        }
        println!( "\n");
    }
}